tests/functional/spec/same-survey-diff-month.spec.js

```diff
@@ -0,0 +1,73 @@
+import assert from 'assert'
+import chai from 'chai'
+import {getRandomString, startQuestionnaire} from '../helpers'
+import reportingPeriod from '../pages/surveys/rsi/0102/reporting-period.page'
+import retailTurnoverPage from '../pages/surveys/rsi/0102/retail-turnover.page'
+import internetSalesPage from '../pages/surveys/rsi/0102/internet-sales.page'
+import changeInRetailTurnover from '../pages/surveys/rsi/0102/changes-in-retail-turnover.page'
+import rsiSummaryPage from '../pages/surveys/rsi/0102/rsi-summary.page'
+import devPage from '../pages/dev.page'
+import landingPage from '../pages/landing.page'
+import multipleSurveys from '../pages/multiple-survey.page'
+
+const expect = chai.expect
+
+describe('RSI - Multiple months of one form type being worked on at the same time ', function() {
+
+    it('Given the RSI survey 0102 when a second surveys is open for a different month then the first survey shows the multiple survey page and the 2nd survey continues to work and has the correct data', function() {
+
+        //Given that RSI business survey 0102
+        devPage.open()
+          .setUserId(getRandomString(10))
+          .setCollectionId(getRandomString(3))
```

Comment:
Possibility that both launches will contain the same values because they are random; however unlikely it could cause a false positive. What about prefixing each with something unique e.g. "A" + getRandom...